curator-client/src/main/java/org/apache/curator/ensemble/exhibitor/ExhibitorEnsembleProvider.java

```diff
@@ -301,7 +301,7 @@ else if ( port != thePort )
                     values.putAll(decodeExhibitorList(encoded));
                     done = true;
                 }
-                catch ( Throwable e )
+                catch ( Exception e )
```

Comment:
I don't understand how this is an improvement. Exceptions such as NPE or OOM would be lost with this change. By catching Throwable, those exceptions are logged.